pkg/vpcmodel/externalNetwork.go

```diff
@@ -153,3 +156,22 @@ func isEntirePublicInternetRange(nodes []Node) (bool, error) {
 	}
 	return nodesRanges.Equal(allInternetRagnes), nil
 }
+
+func (g *groupedExternalNodes) String() string {
+	// 1. Created a list of IPBlocks
+	cidrList := make([]string, len(*g))
+	for i, n := range *g {
+		cidrList[i] = n.Cidr()
+	}
+	ipbList, _, err := ipStringsToIPblocks(cidrList)
+	if err != nil {
+		return ""
+	}
+	// 2. Union all IPBlocks in a single one; its intervals will be the cidr blocks or ranges that should be printed, after all possible merges
+	unionBlock := &common.IPBlock{}
+	for _, ipBlock := range ipbList {
+		unionBlock = unionBlock.Union(ipBlock)
+	}
+	// 3. print a list s.t. each element contains either a single cidr or an ip range
+	return strings.Join(unionBlock.ListToPrint(), commaSepartor)
+}
```

Comment:
I think this function should be moved to the file `grouping.go` (where `groupedExternalNodes` is defined).